src/zenml/zen_stores/migrations/versions/389046140cad_data_versioning.py

```diff
@@ -45,15 +46,31 @@ def upgrade() -> None:
             existing_type=sqlmodel.sql.sqltypes.AutoString(),
         )
 
-    # ### end Alembic commands ###
+    # Model Version Artifacts and Runs Tables
+    with op.batch_alter_table(
+        "model_versions_artifacts", schema=None
+    ) as batch_op:
+        batch_op.alter_column(
+            "artifact_id", existing_type=sa.CHAR(length=32), nullable=False
+        )
+        batch_op.drop_column("step_name")
+        batch_op.drop_column("version")
+        batch_op.drop_column("pipeline_name")
+        batch_op.drop_column("name")
+    with op.batch_alter_table("model_versions_runs", schema=None) as batch_op:
+        batch_op.alter_column(
+            "pipeline_run_id", existing_type=sa.CHAR(length=32), nullable=False
+        )
+        batch_op.drop_column("name")
 
 
 def downgrade() -> None:
-    """Downgrade database schema and/or data back to the previous revision."""
-    # ### commands auto generated by Alembic - please adjust! ###
-    with op.batch_alter_table("artifact", schema=None) as batch_op:
-        batch_op.drop_column("has_custom_name")
-        batch_op.drop_column("version_number")
-        batch_op.drop_column("version")
+    """Downgrade database schema and/or data back to the previous revision.
 
-    # ### end Alembic commands ###
+    Raises:
+        NotImplementedError: Downgrade is not implemented for this migration
+            because the migration drops non-nullable columns.
```

Comment:
I think that we can still restore them by looking at other tables. Might not worth effort, but still

Reply:
Yeah we could do that but as you said I don't think it's worth the effort